internal/app/admin/http/handler.go

```diff
@@ -109,52 +109,62 @@ func configDumpHandler(bootstrapConfig *bootstrapv1.Bootstrap) http.HandlerFunc
 	}
 }
 
-// TODO(lisalu): Support dump of matching resources when cache key regex is provided.
+func printCacheEntries(keys []string, cache cache.ReadOnlyCache, w http.ResponseWriter) {
+	resp := marshallableCache{}
+	for _, key := range keys {
+		resource, err := cache.FetchReadOnly(key)
+		if err != nil {
```

Comment:
should be `err == nil` I think

Reply:
or better,
```
if err != nil {
    resp.Cache = append(resp.Cache, resourceToPayload(key, resource)...)
}
```

Reply:
@jessicayuen  err equaling nil means we have no error right (i.e. the fetch returned successfully)? If it returned successfully, we should append the resource.

Reply:
i think you mean this?
```
if err == nil {
    resp.Cache = append(resp.Cache, resourceToPayload(key, resource)...)
}
```

Reply:
yep, derp my mistake. How you had it is fine too.